aurae-ebpf/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "aurae-ebpf"
```

Comment:
Would we only every have a single ebpf program, or would we have 1 per task. If it is the latter, then the crate name should be more specific.

Reply:
So the way I was thinking about it is having a single binary that would include all the eBPF programs. The programs are named via this [aya attribute](https://github.com/JeroenSoeters/aurae/blob/posix-signals-api/aurae-ebpf/src/main.rs#L17). The `BpfLoader` will then [load these programs](https://github.com/JeroenSoeters/aurae/blob/posix-signals-api/auraed/src/ebpf/loader.rs#L42) from the elf file `aurae-ebpf` into the kernel and [attach them to the correct trace event](https://github.com/JeroenSoeters/aurae/blob/posix-signals-api/auraed/src/ebpf/loader.rs#L48).  I will rename this `name` parameter on `load_tracepoint` to `prog_name` to make it more explicit that it's referring to an actual eBPF program instead of being some random name.